packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -507,3 +509,15 @@ export default function pluginContentBlog(
     },
   };
 }
+
+pluginContentBlog.validateOptions = (opt: Object): ValidationResult => {
```

Comment:
I think the appropriate input type is "unknown", as we don't know exactly what the user will provide us. He could as well give us a number/string or whatever

Reply:
Not sure what is better between:
- `pluginContentBlog.validateOptions`
- `export function validateOptions()`

Reply:
I think we can do something simpler that hides some impl details to the user, like:


```
export function validateOptions({options,validate}) {
  const validOptions = validate(PluginOptionSchema,options)
  return validOptions;
}
```

This way the user wouldn't have to deal with yup error handling logic. He can just throw a simple error if he wants (and you can test if error thrown is a Validation error or not)

Reply:
@slorber fixed